mesonbuild/build.py

```diff
@@ -241,7 +241,8 @@ def __init__(self, environment: environment.Environment):
         self.project_name = 'name of master project'
```

Comment:
Commit messages typos: `which fix this. This goal of this patch` should be `which fixes this. The goal of this patch`. Also a full stop missing after the first paragraph, if you like.